src/test/regress/Makefile

```diff
@@ -10,6 +10,10 @@ ifndef MAJORVERSION
 MAJORVERSION := $(basename $(VERSION))
 endif
 
+ifeq ($(TESTHLL),true)
+HLLEXT := "--load-extension=hll"
```

Comment:
> `undefined_file` is probably a better exception to catch, that'll not trigger for extension script failures and such

That's what I was going to do, then decided to just use `pg_available_extensions` or whatever it's called.